Yoast/wordpress-seo
# Adds support to content analysis for Jetpack Markdown module.

## Summary

This PR can be summarized in the following changelog entry:

* Adds support to content analysis for Jetpack Markdown module.

## Relevant technical choices:

* Created a little JS plugin that parses the markdown in the content before it is analyzed.
* Chose https://github.com/chjj/marked to parse the markdown since it's widely used and lightweight.

## Test instructions

This PR can be tested by following these steps:

* Install Jetpack and enable development mode, see https://jetpack.com/support/development-mode/
* Enable the markdown module under _settings > writing_
* Reproduce issues listed below and find this PR solves them.

Fixes #4933 
Fixes #3406
